frontend/src/modules/history/components/Translation.js

```diff
@@ -199,36 +233,9 @@ export default class Translation extends React.Component<Props, State> {
                         />
                     </div>
                     <menu className='toolbar'>
-                    { index === 0 ? null :
-                        this.state.isDiffVisible ?
-                            // Hide Diff
-                            <Localized
-                                id='history-translation-hide-diff'
-                                attrs={{ title: true }}
-                            >
-                                <button
-                                    className='toggle-diff hide'
-                                    title='Hide diff against the currently active translation'
-                                    onClick={ this.toggleDiff }
-                                >
-                                    { 'Hide diff' }
-                                </button>
-                            </Localized>
-                            :
-                            // Show Diff
-                            <Localized
-                                id='history-translation-show-diff'
-                                attrs={{ title: true }}
-                            >
-                                <button
-                                    className='toggle-diff show'
-                                    title='Show diff against the currently active translation'
-                                    onClick={ this.toggleDiff }
-                                >
-                                    { 'Show diff' }
-                                </button>
-                            </Localized>
-                    }
+
+                    { index > 0 ? this.renderDiffToggle() : null }
```

Comment:
I would even go further and put only `this.renderDiffToggle(index)` here, and do the `if index` test in that method. But it's a nit, if you prefer this way it's fine by me.